addons/functions/functions/fnc_vehicleAction.sqf

```diff
@@ -133,10 +133,10 @@ if (hasInterface) then {
 		true,
 		true,
 		"",
-		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1)", // _target, _this, _originalTarget
+		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1) && player == driver _target", // _target, _this, _originalTarget
```

Comment:
do use `ACE_player` or `call CBA_fnc_currentUnit` so feature works for Zeus while using remote control

Reply:
oooh or use `_this` , apparently it's the caller / the unit the action is shown to

https://community.bistudio.com/wiki/addAction

Reply:
```suggestion
		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1) && (call CBA_fnc_currentUnit) == driver _target", // _target, _this, _originalTarget
```

Reply:
```suggestion
		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1) && _this == driver _target", // _target, _this, _originalTarget
```

(purely from what I read in the documentation, I have not tested this)